verity/src/helm/templates/deployment.yaml

```diff
@@ -13,31 +13,51 @@ spec:
       maxUnavailable: 0
   selector:
     matchLabels:
-      app: {{ template "verity.fullname" . }}
+      {{- include "verity.selectorLabels" . | nindent 6 }}
   replicas: {{ .Values.startReplicas }}
   template:
     metadata:
       labels:
-        app: {{ template "verity.fullname" . }}
+        {{- include "verity.labels" . | nindent 8 }}
```

Comment:
why do you change `verity.fullname` to `verity.labels` in this case but
`verity.fullname` to `verity.selectorLabels` in the previous piece of code? Is that correct? Do both `verity.labels` and `verity.selectorLabels` have always the same value?